### ISSUE TITLE
Fix Company Management & Display Issues

### ISSUE THREAD

**Issue by esdraelon** [amtgard/ORK3#112]
Everything about using companies is not right.  Getting out of them.  False association on the user page.

It's a tragedy.

**Comment by Glenalth** [comment]
The company listing in the Player Details panel can probably go away at this point since the single company restriction is no longer a rule.

**Comment by esdraelon** [comment]
Yeah, I think the difference between "crap that made the rulebook" and "how
the social function of the game" works still exists, though.

**Comment by Glenalth** [comment]
It exists for people that follow traditions, but the newbies are already doing what newbies do.

**Comment by Hunteil** [comment]
^Some Companies still enforce the single company thing. Old habits die hard.

However the profile box for the Households and Companies is still a good feature in my opinion. It's a nice way to find out who is in your household/company and you didn't know it.
